appengine/loopback/common/models/message.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function(Message) {
+  Message.greet = function(msg, cb) {
+    process.nextTick(function() {
+      msg = msg || 'hello';
+      cb(null, 'Sender says ' + msg + ' to receiver');
```

Comment:
These files are generated by the loopback app scaffolder. I'm not going to manually modify any of them.